packages/typespec-test/test/loadtesting_modular/generated/typespec-ts/src/testRunOperations/api/operations.ts

```diff
@@ -1495,19 +1905,13 @@ export function _stopTestRunSend(
   context: Client,
   testRunId: string,
   options: StopTestRunOptionalParams = { requestOptions: {} },
-): StreamableMethod<
-  LoadTestRunStopTestRun200Response | LoadTestRunStopTestRunDefaultResponse
-> {
+): StreamableMethod<undefined> {
   return context
-    .path("/test-runs/{testRunId}:stop", testRunId)
+    .path("/test-runs/{testRunId}:stopTestRun", testRunId)
```

Comment:
The referred path in RLC and modular is different and we could reproduce this in [playground](https://cadlplayground.z22.web.core.windows.net/cadl-azure/?c=aW1wb3J0ICJAdHlwZXNwZWMvaHR0cCI7CtIZcmVzdNUZdmVyc2lvbmluZ8wfYXp1cmUtdG9vbHMvyCstxhVjb3Jl3yvIK3Jlc291cmNlLW1hbmFnZXLfN2MtY2xpZW50LWdlbmVyYXRvcshtCnVzaW5nIEHENC5Db3JlO9ESLlRyYWl0c8gZVHlwZVNwZWMuUmVzdNEVSHR0cNEVVukBCsgbT3BlbkFQSc9t5QCkR%2BgAo8V9OwoKLy8gLcQBIEhlcmUgaXMgbWFpbi50c3DFFwpAc2VydmljZSh7CiAgdGl0bGU6ICJDb250b3NvIFdpZGdldCBN5wEmLAp9KQpA5wGZZWQoxycuxifHJugAr3MpCm5hbWVzcGFjZSDVKuQAnSoqIFRoySDPcSDnAJwgx3MuICovCmVudW0gyGQg5AC2xEXHESAyMDIyLTA4LTMxxCwgIEB1c2VEZXBlbmRlbmN5KOsBl8hDLnYxXzBfUHJldmlld18yKQogIGDJRjBgLAp9CgpAZG9jKCJGaWxlIGluZm8iKQpA6AJDKCJmaWxlcyIpCm1vZGVsIMQlSW5mb%2BUAmsY4TmFtZSBvZiB0aGUgxC0uIsRlQGtlecQHdmlzaWJpbGl0xA7EH8QvOiBzdHLlAfF9CgppbnRlcmbkAUdGb8xYU3RvcCB0ZXN0IHJ1biBiecoMSWTHZG9w5AINaW9uSWQoIkxvYWRUZXN0UnVuX8Q8xwzFKnPETGlzIFN0YW5kYXJkUucA30%2FIQXPkArHFE0FjxBA86ADpLCB7fSzpAPfEDj7lALbxAnhjdXN0b21pesVPIGluIOYDVesCi8YR5QKK5AI7OiDsALPmAt8iLOQAreYCFzr2AlflAqbqATTHP%2BoAyeUBQuQA8Mca5ACcRm9vLsQT5AC6&e=%40azure-tools%2Ftypespec-autorest&options=%7B%22linterRuleSet%22%3A%7B%22extends%22%3A%5B%22%40azure-tools%2Ftypespec-azure-rulesets%2Fresource-manager%22%5D%7D%7D).